README.md

```diff
@@ -370,73 +279,20 @@ The `nf-core download` command will download both the pipeline code and the [ins
 If run without any arguments, the download tool will interactively prompt you for the required information.
 Each option has a flag, if all are supplied then it will run without any user input needed.
 
-```console
-$ nf-core download
-
-                                          ,--./,-.
-          ___     __   __   __   ___     /,-._.--~\
-    |\ | |__  __ /  ` /  \ |__) |__         }  {
-    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
-                                          `._,._,'
-
-    nf-core/tools version 2.2
-
-
-Specify the name of a nf-core pipeline or a GitHub repository name (user/repo).
-? Pipeline name: rnaseq
-? Select release / branch: 3.0  [release]
-
-In addition to the pipeline code, this tool can download software containers.
-? Download software container images: singularity
-
-Nextflow and nf-core can use an environment variable called $NXF_SINGULARITY_CACHEDIR that is a path to a directory where remote Singularity
-images are stored. This allows downloaded images to be cached in a central location.
-? Define $NXF_SINGULARITY_CACHEDIR for a shared Singularity image download folder? [y/n]: y
-? Specify the path: cachedir/
-
-So that $NXF_SINGULARITY_CACHEDIR is always defined, you can add it to your ~/.bashrc file. This will then be autmoatically set every time you open a new terminal. We can add the following line to this file for you:
-export NXF_SINGULARITY_CACHEDIR="/path/to/demo/cachedir"
-? Add to ~/.bashrc ? [y/n]: n
-
-If transferring the downloaded files to another system, it can be convenient to have everything compressed in a single file.
-This is not recommended when downloading Singularity images, as it can take a long time and saves very little space.
-? Choose compression type: none
-INFO     Saving 'nf-core/rnaseq
-          Pipeline release: '3.0'
-          Pull containers: 'singularity'
-          Using $NXF_SINGULARITY_CACHEDIR': /path/to/demo/cachedir
-          Output directory: 'nf-core-rnaseq-3.0'
-INFO     Downloading workflow files from GitHub
-INFO     Downloading centralised configs from GitHub
-INFO     Found 29 containers
-Downloading singularity images ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% • 29/29 completed
-```
+![`ORDER=01 nf-core download rnaseq -r 3.0 --outdir nf-core-rnaseq -x none -c none`](docs/images/nf-core-download.svg)
```

Comment:
The `ORDER` prefixes can be removed as soon as https://github.com/ewels/rich-codex/issues/32 is fixed or we use `fake_commands` instead.

Reply:
Now fixed and released ✅